GitUI/BranchTreePanel/RepoObjectsTree.ContextActions.cs

```diff
@@ -245,9 +245,9 @@ private void contextMenu_Opening(object sender, CancelEventArgs e)
 
         private void contextMenu_Opened(object sender, EventArgs e)
         {
-            // Waiting for ContextMenuStrip (as the visual parent of its menu items) to be visible to
+            // Waiting for the ContextMenuStrip (as the visual parent of its menu items) to be visible to
             // toggle (depending on ToolStripItem.Visible) existing separators in between item groups as required.
-            (sender as ContextMenuStrip)?.ToggleSeparators();
+            menuMain.ToggleSeparators();
```

Comment:
Why did this change?

Reply:
To simplify and avoid gas-lighting developers into thinking this to be a generic Opened event handler that is used by context menus other than `menuMain`. Your more generic solution works for me tho!